CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -82,114 +40,99 @@ class HomeViewController: UIViewController {
         profileButton.addTarget(self, action: #selector(profilePressed), for: .touchUpInside)
         view.addSubview(profileButton)
 
-        reachOutButton.backgroundColor = .backgroundOrange
-        reachOutButton.setTitle("Reach out!", for: .normal)
-        reachOutButton.setTitleColor(.white, for: .normal)
-        reachOutButton.titleLabel?.font = ._20CircularStdBold
-        reachOutButton.layer.cornerRadius = reachOutButtonSize.height/2
-        reachOutButton.addTarget(self, action: #selector(reachOutPressed), for: .touchUpInside)
-        view.addSubview(reachOutButton)
-
-        titleLabel.text = "Meet your Pear"
-        titleLabel.textColor = .textBlack
-        titleLabel.font = ._24CircularStdMedium
-        view.addSubview(titleLabel)
-
-        setupConstraints()
+        tabContainerView = UIView()
+        view.addSubview(tabContainerView)
+        tabPageViewController.view.frame = tabContainerView.frame
+        tabContainerView.addSubview(tabPageViewController.view)
+
+        let tabLayout = UICollectionViewFlowLayout()
+        tabLayout.minimumInteritemSpacing = 0
+
+        tabCollectionView = UICollectionView(frame: .zero, collectionViewLayout: tabLayout)
+        tabCollectionView.delegate = self
+        tabCollectionView.dataSource = self
+        tabCollectionView.register(HomeTabOptionCollectionViewCell.self, forCellWithReuseIdentifier: tabCellReuseIdentifier)
+        tabCollectionView.backgroundColor = .white
+        tabCollectionView.clipsToBounds = true
+        tabCollectionView.layer.masksToBounds = false
+        tabCollectionView.layer.cornerRadius = 24
```

Comment:
The rounding of the sides look a little bit off to me, although the Figma file says 24.
<img width="85" alt="image" src="https://user-images.githubusercontent.com/19177346/94884652-0eee9e00-043c-11eb-83b3-ea1c6668ec70.png">

I tried it with cornerRadius=20 and got this
<img width="97" alt="image" src="https://user-images.githubusercontent.com/19177346/94884613-f54d5680-043b-11eb-808e-aba95a80b4df.png">